README.md

```diff
@@ -36,6 +41,16 @@ For a gentle introduction, please take a look at our [Quickstart Guide](https://
 - Avoid duplicative code
 - Everyone loves YAML! ;)
 
+## 📢 Dag-Factory 1.0 Released
+
+Version **1.0** introduces important improvements and breaking changes to support modern Airflow usage.
+
+👉 See the [Migration Guide](https://astronomer.github.io/dag-factory/latest/migration_guide/) to upgrade from earlier versions.
+
+## 🚀 Dag-Factory Supports Airflow 3
+
+DAG-Factory is compatible with **Apache Airflow 3** and supports modern scheduling, and updated import paths.
+
```

Comment:
Do we need this section repeated here? We have it already in the user facing public documentation.

Would be nice to not repeat information to avoid maintenance efforts.